app-typescript/components/Layouts/Panel.tsx

```diff
@@ -61,22 +66,50 @@ class PanelHeader extends React.PureComponent<IPropsPanelHeader> {
             [`side-panel__header--${this.props.color}`]: this.props.color || this.props.color !== undefined,
             'side-panel__header--has-close': this.props.onClose,
         }, this.props.className);
+
         let style = {
-            zIndex: 10 + (this.props.zIndex ? this.props.zIndex : 0),
+            zIndex: this.props.zIndex ? this.props.zIndex : 10,
         };
+
         let defaultTheme = darkColors.includes(this.props.color || '') ? 'dark-ui' : null;
 
         return (
             <div data-theme={this.props.theme || defaultTheme} className={classes} style={style}>
-                {this.props.onClose ?
-                    <a className="icn-btn side-panel__close"
-                        onClick={() => this.props.onClose ? this.props.onClose() : false}>
-                        <Icon name='close-small' />
-                    </a> : null}
-                {this.props.title &&
-                    <div className="side-panel__header-inner">
+                {this.props.title
+                    && <div className="side-panel__header-inner">
                         <h3 className="side-panel__heading">{this.props.title}</h3>
-                    </div>}
+                    </div>
+                }
+                <ButtonGroup
+                    className='side-panel__btn-group'
+                    align='center'
+                    spaces='no-space'
+                >
+                    {this.props.onPin
+                        ? <IconButton
+                            icon='pin'
+                            ariaValue='Pin'
+                            onClick={() => {
+                                if (this.props.onPin) {
+                                    this.props.onPin();
+                                    this.setState({
+                                        rotate: !this.state.rotate,
+                                    });
+                                }
+                            }}
+                            rotate={this.state.rotate ? '90' : undefined}
+                        />
+                        : null
+                    }
+                    {this.props.onClose
+                        ? <IconButton
+                            icon='close-small'
+                            ariaValue='Close'
+                            onClick={this.props.onClose}
+                        />
+                        : null
```

Comment:
Line 104 - check if the onClose function is not null explicitly. 
Add brackets around the `<IconButton />`
You can use `&&` here since the second option is `null`.